pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/SimplePlayerActivity.kt

```diff
@@ -5,36 +5,53 @@
 package ch.srgssr.pillarbox.demo.ui.player
 
 import android.app.PictureInPictureParams
+import android.content.ComponentName
 import android.content.Context
 import android.content.Intent
+import android.content.ServiceConnection
 import android.content.res.Configuration
 import android.os.Build
 import android.os.Bundle
+import android.os.IBinder
+import android.util.Log
 import android.widget.Toast
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
 import androidx.activity.viewModels
 import androidx.annotation.RequiresApi
 import androidx.compose.material.Surface
 import androidx.lifecycle.lifecycleScope
-import androidx.media3.ui.PlayerNotificationManager
-import ch.srgssr.pillarbox.demo.R
 import ch.srgssr.pillarbox.demo.data.DemoItem
 import ch.srgssr.pillarbox.demo.data.Playlist
+import ch.srgssr.pillarbox.demo.service.DemoPlaybackService
 import ch.srgssr.pillarbox.demo.ui.theme.PillarboxTheme
-import ch.srgssr.pillarbox.player.notification.PillarboxNotificationManager
+import ch.srgssr.pillarbox.player.service.PlaybackService
 import kotlinx.coroutines.flow.collectLatest
 
 /**
  * Simple player activity that can handle picture in picture.
  *
+ * It handle basic background playback, as it will stop playback when the Activity is destroyed!
+ * To have pure background playback with good integration from other device like Auto, Wear, etc... we need *MediaSessionService*
+ *
  * For this demo, only the picture in picture button can enable picture in picture.
  * But feel free to call [startPictureInPicture] whenever you decide, for example when [onUserLeaveHint]
  */
 class SimplePlayerActivity : ComponentActivity() {
 
     private val playerViewModel: SimplePlayerViewModel by viewModels()
-    private lateinit var notificationManager: PlayerNotificationManager
+
+    private val serviceConnection = object : ServiceConnection {
+
+        override fun onServiceConnected(name: ComponentName?, service: IBinder?) {
+            val binder = service as PlaybackService.ServiceBinder
+            binder.setPlayer(playerViewModel.player)
```

Comment:
⚠️ **[Android Lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>SyntheticAccessor: Access to `private` method `getPlayerViewModel` of class `SimplePlayerActivity` requires synthetic accessor